cmd/gateway/main.go

```diff
@@ -260,6 +268,48 @@ func (flags GatewayFlags) NewGateway(ctx context.Context, ident *identity.FullId
 	), nil
 }
 
+func (flags GatewayFlags) interactive(cmd *cobra.Command, setupDir string, overrides map[string]interface{}) error {
+	satelliteAddress, err := cfgstruct.PromptForSatelitte(cmd)
+	if err != nil {
+		return Error.Wrap(err)
+	}
+
+	apiKey, err := cfgstruct.PromptForAPIKey()
+	if err != nil {
+		return Error.Wrap(err)
+	}
+
+	encKey, err := cfgstruct.PromptForEncryptionKey()
+	if err != nil {
+		return Error.Wrap(err)
+	}
+
+	overrides["satellite-addr"] = satelliteAddress
+	overrides["api-key"] = apiKey
+	overrides["enc.key"] = encKey
+
+	err = process.SaveConfigWithAllDefaults(cmd.Flags(), filepath.Join(setupDir, "config.yaml"), overrides)
```

Comment:
I thought we don't want to be saving all defaults for better backwards compatibility.

Reply:
That's new to me but we are doing this in Uplink and the story was do what uplink does

Reply:
to use just `SaveConfig` we need to mark some flags with `user` tag to save them in config. We did it for storage node to limit number of flags in config file. It would be good to do the same thing for uplink but I believe its separate task.